WORKSPACE

```diff
@@ -37,7 +37,7 @@ load("//tensorflow:workspace.bzl", "tf_workspace")
 #    # API level that it supports without notice.
 #    # Note that the NDK version is not the API level.
 #    api_level=14)
```

Comment:
#    api_level=14)
this line was missing in my file for some reason
and it is good that you noticed.....
because my build needs it